### PR TITLE
Fix 1068 georef output

### LINKED ISSUE #1068: [FEATURE] Georeference outputs (eg PDF) from composer
Original commit: https://github.com/qgis/QGIS/commit/bae8a0e5e0b8b86f5eeed94c323a4fffac846c46 by nyalldawson

This commit makes composer automatically georeference outputs
(where output format makes this possible, eg TIF and PDF).

The existing option to create a world file has been separated
from the map selection for georeferencing. The new behaviour
is to always georeference outputs, and only create the
separate world file if that option is checked.

### PR DESCRIPTION
**Pull request by SrNetoChan**
Fixes #1068 
Fixes a bad reference that was blocking the build
Small changes to grammar and sphinx syntax

### CODE REVIEW

File: source/docs/user_manual/print_composer/create_output.rst [hunk 1]
```diff
@@ -76,24 +77,30 @@ include the area of that page with items.
    Image Export Options
 
 If you need to export your layout as a **georeferenced image** (e.g., to share
-with other projects), you need to enable this feature under the Composition tab.
-Check |checkbox| :guilabel:`World file on` and choose the map item to use.
+with other projects), you need to enable this feature under the
+:ref:`composer_composition_tab`.
+
+If the output format is a TIFF format, all you need to do is make sure to
```
Comment (yjacolin) [inline source/docs/user_manual/print_composer/create_output.rst]: `is TO make sure` ?

File: source/docs/user_manual/print_composer/overview_composer.rst [hunk 2]
```diff
@@ -403,14 +403,16 @@ Export settings
 ...............
 
 You can define a resolution to use for all exported maps in :guilabel:`Export
-resolution`. This setting can however be overridden each time you are exporting a map.
+resolution`. This setting can, however, be overridden each time you are
+   exporting a map.
```
Comment (DelazJ) [inline source/docs/user_manual/print_composer/overview_composer.rst]: indentation...

**Comment by DelazJ** [comment]
@SrNetoChan be careful when cherry-picking this one to master because:
* the screenshot should remain as before
* and the reference map option is moved under general settings section

**Comment by SrNetoChan** [comment]
@DelazJ regarding the cherry-picking to master, we will need to update the qgis3 screenshot anyway. The the old one does not fit the current GUI on 2.99.

I think I will just cherry pick this commits and assign #1622 to me, From there I will do the necessary changes to QGIS 3.

**Pull request merged**